app/models/fee/misc_fee_type.rb

```diff
@@ -5,7 +5,7 @@ class Fee::MiscFeeType < Fee::BaseFeeType
   AGFS_SUPPLEMENTARY_TYPES = (AGFS_SUPPLEMENTARY_ONLY_TYPES + AGFS_SUPPLEMENTARY_SHARED_TYPES).freeze
   TRIAL_ONLY_TYPES = %w[MIUMU MIUMO].freeze
 
-  default_scope { order(description: :asc) }
+  default_scope { order(Arel.sql("regexp_replace(\"fee_types\".\"description\", '[()]', '', 'g')")) }
```

Comment:
Is the difference in ordering to do with collation? See [this Stack Overflow question.](https://stackoverflow.com/questions/56395711/postgresql-sort-a-character-field-with-special-characters-numbers-and-alphabe)

Reply:
you'd hope so, but I tried collation first and it did not work

Reply:
tried again but none of the following do what i'd expect/want for the unit test
```
default_scope { order(Arel.sql("\"fee_types\".\"description\" COLLATE \"POSIX\""))}
default_scope { order(Arel.sql("\"fee_types\".\"description\" COLLATE \"C\""))}
```
or any of the regional/language collations either

Reply:
it seems to be differences between darwin/alpine and debian/amazon-linux